Readme.md

```diff
@@ -5,43 +5,137 @@
 > 2023**.
 
 Capi is a framework for crafting interactions with Substrate chains. It consists
-of a development server and fluent API, which facilitates multistep, multichain
+of a development server and fluent API, which facilitates multichain
 interactions without compromising either performance or ease of use.
 
-- [Docs &rarr;](https://docs.capi.dev)<br />Guides for Capi developers and
-  pattern library developers
 - [Examples &rarr;](./examples)<br />SHOW ME THE CODE
 - [API Reference &rarr;](https://deno.land/x/capi/mod.ts)<br />A generated API
   reference, based on type signatures and TSDocs.
 
 ## Installation
 
-### [Node.js](https://nodejs.org/)
-
 ```sh
-npm i capi https://capi.dev/frame/wss/rpc.polkadot.io/@latest/pkg.tar
+npm i capi
 ```
 
-### [Deno](https://deno.land/)
+<details >
+<summary>Deno Equivalent</summary>
+<br>
 
 `import_map.json`
 
 ```json
 {
   "imports": {
-    "@capi/polkadot": "https://capi.dev/frame/wss/rpc.polkadot.io/@latest/"
+    "capi": "https://deno.land/x/capi/mod.ts"
   }
 }
 ```
 
+</details>
+
+## Configuration
+
+Create a `capi.config.ts` and specify the chains with which you'd like to
+interact.
+
+```ts
+import { CapiConfig } from "capi"
+
+export const config: CapiConfig = {
+  server: "https://capi.dev/",
+  chains: {
+    polkadot: {
+      url: "wss://rpc.polkadot.io/",
+      version: "v0.9.40",
+    },
+  },
+}
+```
+
+Let's also configure an ephemeral development network.
```

Comment:
I think doing this in two separate steps overcomplicates the readme. Can you combine these into one step, and perhaps add inline comments to explain that one is an ephemeral development network?